internal/cri/server/sandbox_stats_windows.go

```diff
@@ -106,7 +106,7 @@ func convertMetricsToWindowsStats(metrics []*types.Metric, sandbox sandboxstore.
 		// In the case of HostProcess sandbox container we will use the nil value for the statsmap which is used later
 		// otherwise return an error since we should have gotten stats
 		containerStats, ok := containerStatsData.(*wstats.Statistics)
-		//nolint:staticcheck QF1001: could apply De Morgan's law (staticcheck)
+		//nolint:staticcheck
```

Comment:
Having the comment is still useful, but should use this format;

```go
//nolint:staticcheck // QF1001: could apply De Morgan's law (staticcheck)
```


When making that change, could you amend the `fix: QF1001: could apply De Morgan's law (staticcheck)` commit, instead of doing so in a separate commit?